src/runtime/node/tls/index.ts

```diff
@@ -0,0 +1,46 @@
+import type tls from "node:tls";
+import { notImplemented, notImplementedClass } from "../../_internal/utils";
+import { TLSSocket } from "./tls-socket";
+import { Server } from "./server";
+import { SecureContext } from "./secure-context";
+import * as constants from "./constants";
+
+export * from "./constants";
+export { TLSSocket } from "./tls-socket";
+export { Server } from "./server";
+export { SecureContext } from "./secure-context";
+
+export const connect: typeof tls.connect = function () {
```

Comment:
these anonymous functions might be a pain in the butt to deal with when debugging. Could you please name them (`function connect ...`) or export them as functions rather than `const`s? Thanks

Reply:
Although v8 uses some dark magic to infer function name from variables, i fully agree explicitly naming functions can help 

---

```js
> const x = function() { throw new Error('a') }
undefined
> x()
Uncaught Error: a
    at x (REPL1:1:30)
> 
```

Reply:
~> https://github.com/unjs/unenv/issues/222